DummyAiHttpServer/README.md

```diff
@@ -0,0 +1,28 @@
+# Simple http server that listens on port 7861 and endpoint /sdapi/v1/txt2img
```

Comment:
Please move whole `DummyAiHttpServer` to `examples` directory. We have to many top level projects